scripts/test-config-cuda.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+set -e
+
+# set log location from command invokation
+LOG_LOC=$1
+TEST_FAIL=false
+
+#!/bin/bash
+set -e
+
+# set log location from command invokation
+LOG_LOC=$1
+TEST_FAIL=false
```

Comment:
Is this part necessary?

Reply:
You're right, I'll move it to `scripts/tests`. That seems like a better home for it.  I'll put the JSON file with the tested version combinations in there too.

Thanks for catching that duplicated code! I've removed it.